### PR TITLE
Re-run with all CRAN versions

### PR DESCRIPTION
**Pull request by topepo**
Except `DT` 🙄

### CODE REVIEW

**Comment by topepo** [comment]
Um, actually, don't merge yet. I realize that CRAN tibble was used.

**Comment by topepo** [comment]
It's good now

File: content/find/all/index.html [hunk 1]
```diff
@@ -4,6 +4,7 @@
 weight: 1
 ---
 
+<script src="/rmarkdown-libs/header-attrs/header-attrs.js"></script>
```
Comment (apreshill) [inline content/find/all/index.html:7]: I just removed this yesterday and semi-blamed it for the weird table rendering. It looks like the DT tables render fine in your deploy preview, but I'm curious if you deleted all the `find/*.html` files then re-rendered? For me, this js script dis-appeared.

Comment (topepo) [reply content/find/all/index.html:7]: I'm pretty sure that I did not delete them. I'll merge and check the tables.

**Comment by topepo** [comment]
This has gotten way out of sync with the master branch. I'm going to close this PR and re-render again with the small changes made here. I'm worried that things might get borked.

**Pull request closed without merging**